lib/processes/proxy/proxy.js

```diff
@@ -30,7 +30,7 @@ function trimHash(url) {
 }
 
 function normalizeOptions(options) {
-  options.url = trimHash(options.url);
+  options.url = trimHash(options.url || '/');
```

Comment:
The problem is that `trimHash` will throw and kill the proxy if `options.url` is undefined. So the `||` has to be inside of the call.